.teamcity/Gradle_Check/model/BucketProvider.kt

```diff
@@ -239,28 +239,28 @@ class LargeSubprojectSplitBucket(val subproject: GradleSubproject, val number: I
         val testClasses = classes.map { it.toPropertiesLine() }
         val action = if (include) "include" else "exclude"
         val unixScript = """
-mkdir -p build
-rm -rf build/*-test-classes.properties
-cat > build/$action-test-classes.properties << EOL
+mkdir -p test-splits
+rm -rf test-splits/*-test-classes.properties
```

Comment:
Should we delete the whole `test-splits` directory before adding the new `-test-classes.properties`?

Reply:
No need I think...